src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private final int index = new Random().nextInt(Colors.values().length);
+
     public String getRandomColor() {
-        return null;
+        return Colors.values()[index].toString();
```

Comment:
read `checklist.md` about using `.toString()` or `.name()` with enums, and make corresponding fix